easybuild/easyconfigs/p/python-igraph/python-igraph-0.9.8-foss-2021b.eb

```diff
@@ -17,7 +17,7 @@ dependencies = [
     ('Python', '3.9.6'),
     ('igraph', '0.9.5'),
     ('PyCairo', '1.20.1'),
-    ('Clang', '12.0.1'),
+    ('Clang', '13.0.1'),
```

Comment:
@Wentzell Can you clarify why this change was made?

This will cause trouble when this easyconfig is combined with others from the same generation that depend on Clang/12.0.1, like `pocl-1.8-GCC-11.2.0.eb`

Reply:
This change was made because two variants of Clang dependencies were found in the dependency resolution (see [log](https://github.com/easybuilders/easybuild-easyconfigs/runs/5684084504?check_suite_focus=true#step:8:146))

```
found 2 variants of 'Clang' dependency in easyconfigs using '2021b' toolchain generation
* version: 12.0.1; versionsuffix:  as dep for set(['leidenalg-0.8.8-foss-2021b.eb', 'python-igraph-0.9.8-foss-2021b.eb', 'scanpy-1.8.2-foss-2021b.eb'])
* version: 13.0.1; versionsuffix:  as dep for set(['TRIQS-tprf-3.1.0-foss-2021b.eb', 'TRIQS-cthyb-3.1.0-foss-2021b.eb', 'TRIQS-3.1.0-foss-2021b.eb', 'TRIQS-dft_tools-3.1.0-foss-2021b.eb'])
```

TRIQS depends on Clang-13 through the [Clang-Python-bindings-13.0.1](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/c/Clang-Python-bindings/Clang-Python-bindings-13.0.1-GCCcore-11.2.0.eb) dependency. The reason we chose 13 here is that TRIQS itself requires Clang v13+ to compile.

I checked at the time that I created the PR and `python-igraph` was the only `GCC-11.2.0` and `foss-2021b` easyconfig to explicitly list Clang as a dependency, so I updated this.

Would it be an option to update also the `pocl` or would this lead to other potential problems?

Reply:
@boegel Please let me know how to proceed! It would be great to get this merged for the next easybuild release.